software/spmd/interrupt_tests/Makefile

```diff
@@ -23,13 +23,13 @@ endif
 %_run: cov_setup
 	mkdir -p $@
 	cp $*.S $@/main.S
-	$(MAKE) -C $@ -f ../Makefile main.run CM_NAME=$* 2>&1 | /usr/bin/tee $@/$@.log
+	$(MAKE) -C $@ -f ../Makefile main.run CM_NAME=$* COV_DIR=$(abspath ./coverage) 2>&1 | /usr/bin/tee $@/$*.run.log
```

Comment:
LOG_FILES needs to change also. *_run.log -> *.run.log

Reply:
Modified LOG_FILES and SCRAPE-TAIL to reflect the requested change.